### PR TITLE
Tweak contribution guidelines

### PR DESCRIPTION
**Pull request by ziadoz**
I've tweaked the contribution guidelines a bit. The points remain mostly the same, I've just tweaked the wording and laid things out more clearly.

I did remove the "self promotion" point. I recall the intention of this was to avoid brand new projects being suggested. I've replaced it with a mention that projects should ideally be established and mature.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,27 +1,33 @@
 # Contribution Guidelines
-Unfortunately, not every library, tool or framework can be considered for inclusion. The aim of Awesome PHP is to be a concise list of noteworthy and interesting software written in modern PHP. Therefore, suggested software should:
+The aim of Awesome PHP is to be a concise list of the best packages, frameworks, tools and software the PHP ecosystem has to offer. Unfortunately, this means not everything can or should be included. Therefore, suggested software should ideally be one of the following:
```
Comment (alexkart) [inline CONTRIBUTING.md:2]: - ~~The aim of Awesome PHP is to~~ Awesome PHP aims to 
- ~~tools and~~ tools, and
- ~~suggested software~~ the suggested software

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
-2. Well known or discussed within the PHP community
-3. Be unique in its approach or function
-4. Fill a niche gap in the market
+* Widely known or recognised within the PHP community
+* Established and mature
+* Unique in its approach
+* Fulfill a niche gap in the market
 
-Self-promotion is frowned upon, so please consider seriously whether your project meets the criteria before opening a pull request, otherwise it may be closed without being reviewed.
+Where applicable, preference will be give to software that:
```
Comment (alexkart) [inline CONTRIBUTING.md:9]: ~~be give~~ be given

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
-* Use the following format for libraries: \[LIBRARY\]\(LINK\) - DESCRIPTION.
-* Prefix duplicate library names with their vendor or namespace followed by a space: Foo\Bar would be Foo Bar.
-* New categories, or improvements to the existing categorisation, are always welcome.
-* Please keep descriptions short, simple and unbiased. No buzzwords or marketing jargon please.
-* End all descriptions with a full stop/period.
-* Check your spelling and grammar.
-* Make sure your text editor is set to remove trailing whitespace.
-* Your entry has been added alphabetically within the category.
+* Your suggestion will be closed if it has been previously rejected or is a duplicate
+* Write a meaningful pull request description which includes a link to the project
```
Comment (alexkart) [inline CONTRIBUTING.md:22]: ~~which includes~~ that includes

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,27 +1,33 @@
...
-* Please make an individual commit for each suggestion in a separate pull request.
-* Use the following format for libraries: \[LIBRARY\]\(LINK\) - DESCRIPTION.
-* Prefix duplicate library names with their vendor or namespace followed by a space: Foo\Bar would be Foo Bar.
-* New categories, or improvements to the existing categorisation, are always welcome.
-* Please keep descriptions short, simple and unbiased. No buzzwords or marketing jargon please.
-* End all descriptions with a full stop/period.
-* Check your spelling and grammar.
-* Make sure your text editor is set to remove trailing whitespace.
-* Your entry has been added alphabetically within the category.
+* Your suggestion will be closed if it has been previously rejected or is a duplicate
```
Comment (alexkart) [inline CONTRIBUTING.md:21]: What if a package or library was previously unpopular or didn’t meet certain criteria, but now it does and can be added to the list?

Comment (ziadoz) [reply CONTRIBUTING.md:21]: @alexkart I think we can use our own judgement on this.

**Review by alexkart — APPROVED** [review]

**Pull request merged**